Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Setup node environment
+FROM node:18-alpine
+# Create app directory
+WORKDIR /app
+# Copy package.json to /app directory
+COPY package.json /app
+# Install dependencies
+RUN npm install
+RUN npm rebuild esbuild
```

Comment:
Why do you have to run this command? Can you add a comment for this?

Reply:
This command was added due to this issue: https://github.com/evanw/esbuild/issues/1223

It is only an issue when building locally and locally with Docker. This issue would not affect a Docker container on an AWS instance since it will not be built both locally and through Docker.